src/tests/electron/setup/timeouts.ts

```diff
@@ -6,3 +6,6 @@ export const DEFAULT_ELECTRON_TEST_TIMEOUT_MS = 20000;
 
 // How long to wait for an element to be visible
 export const DEFAULT_WAIT_FOR_ELEMENT_TO_BE_VISIBLE_TIMEOUT_MS = 5000;
+
+// How long to wait for an element to have (or not have) a class
+export const DEFAULT_WAIT_FOR_ELEMENT_CLASS_TIMEOUT_MS = 500;
```

Comment:
This is probably a little ambitious for reliable IPC to a browser on a constrained build agent. Personally I'd lean towards just reusing DEFAULT_WAIT_FOR_ELEMENT_TO_BE_VISIBLE_TIMEOUT_MS

Reply:
Great! I was debating about what value to use here. Glad you mentioned this. Updated.